subprojects/docs/src/samples/userguide/antMigration/fileDeps/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+    java
+}
+
+// tag::file-deps[]
+repositories {
+    flatDir {
+        name = "libs dir"
+        dir(file("libs"))  // <1>
+    }
+}
+
+dependencies {
+    implementation(files("libs/our-custom.jar"))  // <2>
+    implementation(":log4j:1.2.8")     // <3>
+    implementation(":commons-io:2.1")  // <3>
+}
+// end::file-deps[]
+
+// tag::retrieve-deps[]
+tasks.create<Copy>("retrieveRuntimeDependencies") {
+    into("$buildDir/libs")
+    from(configurations.runtimeClasspath)
+}
+// end::retrieve-deps[]
+
+// tag::properties[]
+val tmpDistDir by extra { file("$buildDir/dist") }
+
+tasks.create<Jar>("javadocJar") {
+    from(tasks.getByName("javadoc"))  // <1>
+    classifier = "javadoc"
+}
+
+tasks.create<Copy>("unpackJavadocs") {
+    from(zipTree(tasks.getByName<Jar>("javadocJar").archivePath))  // <2>
+    into(tmpDistDir)  // <3>
+}
```

Comment:
:o: this could use a `tasks {}` block to avoid repeating `tasks.` many times like the `importBuild` sample below